integration_tests/testsuite/test_custom.py

```diff
@@ -14,22 +14,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import re
 import json
 import logging
+import requests
 import unittest
 import urlparse
 
 import test_util
 
 
 class TestCustom(unittest.TestCase):
+    """ This TestCase fetch a configuration from the endpoint '/custom' describing
+        a series of tests, then run each of them and report their results.
+
+        In the case where a test of the series fail, this TestCase will be
+        considered as failed.
+
+        The specification for the custom tests can be found at:
+        https://github.com/GoogleCloudPlatform/runtimes-common/tree/master/integration_tests#custom-tests
+    """
 
     def __init__(self, url, methodName='runTest'):
         self._base_url = url
         self._url = urlparse.urljoin(url, test_util.CUSTOM_ENDPOINT)
         unittest.TestCase.__init__(self)
 
     def runTest(self):
+        """ Retrieve the configuration for the custom tests and launch the tests.
+
+        :return: None.
```

Comment:
Would you mind using the Google style guide format for method and class comments?

https://google.github.io/styleguide/pyguide.html#Comments